contributing.md

```diff
@@ -2,8 +2,34 @@
 
 ## Prepare the development environment
 
-### Install the Rust toolchain
+Install [Python 3.9](https://www.python.org/downloads/).
 
-### Setup the Rust toolchain
+If you are using a shell capable of running [Bash](https://www.gnu.org/software/bash/) scripts, run the following from the project 
+root directory:
+```shell
+$ source ./activate-venv.sh
+```
+
+Otherwise, follow these manual steps using this
+[guide](https://docs.python.org/3.9/tutorial/venv.html?highlight=package):
+1. Create virtual environment for your project.
+2. Activate project's virtual environment.
+3. Install all project dependencies in the project's virtual environment using `toolchain-requirements.txt`.
 
 ## Build-related commands
+
+If you are using a shell capable of running Bash scripts, run from the project root directory:
+
+| &#x23; | Command                                             | Description                                                                                                                                                                                                                                                                                                                                   |
```

Comment:
I'd like to see what this table looks like in your IDE editor. The bundled Markdown plugin is supposed to help with editing Markdown tables.